tests/wpt/urltestdata.json

```diff
@@ -5950,6 +5950,20 @@
     "search": "",
     "hash": ""
   },
+  {
+    "input": "../package.json",
+    "base": "file:///Users/package.json",
```

Comment:
Can you make this changes to ada url test data json file? The changes to this file will be overriden after the next wpt test update script.